pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -273,6 +277,24 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 	return f, nil
 }
 
+// setDefaultPluginSetIfNeed will set default enable plugins if we need it and it is empty.
+func setDefaultPluginSetIfNeeded(plugins *config.Plugins) *config.Plugins {
+	if plugins == nil {
+		plugins = &config.Plugins{}
+	}
+
+	if plugins.QueueSort == nil {
+		plugins.QueueSort = &config.PluginSet{}
+	}
+	if len(plugins.QueueSort.Enabled) == 0 {
+		plugins.QueueSort.Enabled = append(plugins.QueueSort.Enabled,
+			config.Plugin{Name: frameworkplugins.DefaultQueueSortPluginName})
```

Comment:
If we use `k8s.io/kubernetes/pkg/scheduler/framework/plugins/queuesort`.`Name`( "default-queue-sort-plugin"), there will be an import cycle(note that `k8s.io/kubernetes/pkg/scheduler/framework/plugins/queuesort` has imported `k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1`), so I introduces `frameworkplugins.DefaultQueueSortPluginName` to resolve it.